src/components/Retooling/index.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+import './styles.postcss';
+import React from 'react';
+
+class Retooling extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  getImages() {
+    return [ 'infants', 'household', 'cotton', 'kinitting', 
+      'babyfood', 'blanket', 'layette', 'woolensuitting', 'standardfood'
+    ];
+  }
+
+  getTitles() {
+    return [
+      'Infant Food Package',
+      'Household Linen Package',
+      'Cotton Package',
+      'Knitting Wool Package',
+      'Baby Food Package',
+      'Blanket Package',
+      'Layette Package',
+      'Woolen Suiting Package',
+      'Standard Food Package'
+    ];
+  }
+
```

Comment:
Could we try to do this with an object?
I think it would be easier with something like:
items =[
  {  title: 'my title',  image: 'my-image' },
  {  title: 'my title 2',  image: 'my-image-2' }
]

This way, we'd avoid order problems.